### ISSUE TITLE
Additional creep checks cannot be cancelled

### ISSUE THREAD

**Issue by jzy-chitong56** [SMUnlimited/AMAI#260]
This check is mainly to avoid certain maps,creep and mine The distance is too far,When the system checks, it is assumed that there are no monsters here

but in reality, when build an expansion,creep Can attack

Although this type of map may not be many, it does exist

**Comment by SMUnlimited** [comment]
Yes, the distance needs to be large enough to cover this but not too large that invalid creep groups are picked up. This happens very early now so its unlikely creeps have been tricked away from the locations at time of check.

**Comment by jzy-chitong56** [comment]
Some maps are initially placed at a large distance, which was intentionally set by the creator

**Comment by jzy-chitong56** [comment]
(4)Avalanche_LV
look this map 's exp , you will konw

**Comment by jzy-chitong56** [comment]
(4)Avalanche_LV

The size of the grille is large ， the distance will let AI think here no creep 
Even if there are no issues here, it cannot be ruled out that other maps or map authors may encounter this situation

So I designed an additional check
And I insist that it cannot be deleted

![image](https://github.com/SMUnlimited/AMAI/assets/48715223/98238e52-2241-4cfe-b9a4-e6bc0aa1a9af)
